server.js

```diff
@@ -98,10 +99,6 @@ if (AppConstants.NODE_ENV === "heroku") {
   SCRIPT_SOURCES.push(PONTOON_DOMAIN);
   STYLE_SOURCES.push(PONTOON_DOMAIN);
   FRAME_ANCESTORS.push(PONTOON_DOMAIN);
-  app.use(helmet.frameguard({
-    action: "allow-from",
-    domain: PONTOON_DOMAIN,
-  }));
```

Comment:
Don't we need this anymore?

Reply:
helmet dropped `frameguard`, and observatory.mozilla.org looks at CSP `framce-ancestors` for this check now.